cfg/common.go

```diff
@@ -167,6 +167,28 @@ type Common struct {
 
 	Traces               Traces `yaml:"traces"`
 	PrintErrorStackTrace bool   `yaml:"printErrorStackTrace"`
+
+	// DeterministicBackend sets a backend weight for each metrics in render
```

Comment:
I think it should be either `each metric` or `all metrics`.

Reply:
thanks, fixed now.